en/docs/administer/product-administration/monitoring/jmx-based-monitoring.md

```diff
@@ -8,24 +8,24 @@ JMX is enabled in WSO2 products by default, which ensures that the JMX server st
 
 ### Configuring JMX ports for the server
 
-The default JMX ports (RMIRegistryPort and the RMIServerPort) are configured in the `carbon.xml` file (stored in the `<PRODUCT_HOME>/repository/conf` directory) as shown below. If required, you can update these default values.
+The default JMX ports (RMIRegistryPort and the RMIServerPort) can be changed by adding the below config into 
```

Comment:
below config => following config

Reply:
What is the difference between these two?